service/src/main/java/uk/gov/hmcts/reform/fpl/service/sdo/ListAdminEventNotificationDecider.java

```diff
@@ -13,22 +13,27 @@
 public class ListAdminEventNotificationDecider {
 
     public Optional<ListAdminEvent> buildEventToPublish(CaseData caseData) {
-        ListAdminEvent event = null;
 
-        GatekeepingOrderEventData gatekeepingOrderEventData = caseData.getGatekeepingOrderEventData();
+        final GatekeepingOrderEventData gatekeepingOrderEventData = caseData.getGatekeepingOrderEventData();
         if (Objects.nonNull(gatekeepingOrderEventData) && gatekeepingOrderEventData.isSentToAdmin()) {
-            ListAdminEvent.ListAdminEventBuilder listAdminEventBuilder = ListAdminEvent.builder()
+            final ListAdminEvent.ListAdminEventBuilder listAdminEventBuilder = ListAdminEvent.builder()
                 .caseData(caseData)
                 .isSentToAdmin(gatekeepingOrderEventData.isSentToAdmin())
                 .sendToAdminReason(gatekeepingOrderEventData.getSendToAdminReason());
 
-            StandardDirectionOrder standardDirectionOrder = caseData.getStandardDirectionOrder();
+            final StandardDirectionOrder standardDirectionOrder;
+            if (Objects.nonNull(caseData.getGatekeepingOrderRouter())) {
+                standardDirectionOrder = caseData.getStandardDirectionOrder();
+            } else {
+                standardDirectionOrder = caseData.getUrgentDirectionsOrder();
```

Comment:
similar to my comment on the other PR, standardDirectionOrder maybe sounds like it's not UDO